# Service to Set the Zone of the robot
# Mode - Name in RAPID - Linear  - Orientation
#   0        fine         0 mm        0°
#   1         z0          0.3 mm      0.03°  <- Default and recommended value.
#   2         z1          1 mm        0.1°
#   3         z5          5 mm        0.8°
#   4         z10         10 mm       1.5°

int64 mode
---
int64 ret
string msg
